db/host_id.hh

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2015-present ScyllaDB
+ *
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "utils/UUID.hh"
+
+using host_id = utils::tagged_uuid<struct host_id_tag>;
```

Comment:
We have a lot of variable named host_id. Is it better to make the type host_id_t?

Reply:
If it would be a structure, not only the identifier than I'd say it warrants to be called host_id_type, but it's an integral identifier, hence the type name.

Also, the convention was inherited from many other identifier types that are defined this way - starting from counter_id and all raft tagged id's, then table_id, query_id, etc. and we have several more to come, like paxos_ballot_id, streaming_plan_id, repair_id? and more

Reply:
ok